nasa/harmony
# HARMONY-1872: Add `skipLibCheck` to harmony service build to try to fix issues

## Jira Issue ID
HARMONY-1872

## Description
Yes, I named this branch after the wrong ticket, this is a fix for 1872, not 1887.  It just adds `skipLibCheck: true` to the Harmony service image build `tsconfig.build.json` file to get around a TypeScript error caused by one of the updated libraries.

## Local Test Steps
1. Make sure queries still work in dev mode. This should not be an issue since the `tsconfig.build.json` file is only used to build the service images.
2. Build the images and test Harmony in a Box

## PR Acceptance Checklist
